core/src/main/scala/ai/h2o/sparkling/backend/PartitionStatsGenerator.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ai.h2o.sparkling.backend
+
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.Row
+
+/**
+  * Goes over RDD partitions counting records and checking if given set of columns has constant values
+  */
+private[backend] object PartitionStatsGenerator {
+
+  def getPartitionStats(
+      rdd: RDD[Row],
+      maybeFeatureColumnsForConstantCheck: Option[Seq[String]] = None): PartitionStats = {
```

Comment:
nit: I would maybe use more generic expression for the variable. feature columns are more related to ml, conversions are more generic.